openslides/motions/static/js/motions/site.js

```diff
@@ -661,25 +661,81 @@ angular.module('OpenSlidesApp.motions.site', ['OpenSlidesApp.motions', 'OpenSlid
 .controller('MotionListCtrl', [
     '$scope',
     '$state',
+    '$http',
     'ngDialog',
     'MotionForm',
     'Motion',
     'Category',
     'Tag',
     'Workflow',
     'User',
-    function($scope, $state, ngDialog, MotionForm, Motion, Category, Tag, Workflow, User) {
+    'Config',
+    'Agenda',
+    function($scope, $state, $http, ngDialog, MotionForm, Motion, Category, Tag, Workflow, User, Config, Agenda) {
         Motion.bindAll({}, $scope, 'motions');
         Category.bindAll({}, $scope, 'categories');
         Tag.bindAll({}, $scope, 'tags');
         Workflow.bindAll({}, $scope, 'workflows');
         User.bindAll({}, $scope, 'users');
         $scope.alert = {};
 
+        // minimum of supporters, if 0 the function is disabled
+        $scope.min_supporters = Config.get('motions_min_supporters').value;
+
         // setup table sorting
         $scope.sortColumn = 'identifier';
         $scope.filterPresent = '';
         $scope.reverse = false;
+        $scope.stateFilter = [];
```

Comment:
You have three times nearly exact code pieces of about 15 lines, so I would say we should follow the DRY principle here.